pyembed/src/conversion.rs

```diff
@@ -30,11 +37,11 @@ const SURROGATEESCAPE: &[u8] = b"surrogateescape\0";
 /// The optional encoding is the name of a Python encoding. If not used,
 /// the default system encoding will be used.
 #[cfg(target_family = "unix")]
-pub fn cpython_osstr_to_pyobject(
-    py: cpython::Python,
+pub fn osstr_to_pyobject<'p>(
```

Comment:
PyO3 has conversions from `OsStr` and `Path` to Python objects. These may potentially be usable for you instead of the ffi you're doing in this file?

Reply:
Yes, it looks like I'll be able to port to your conversion implementations. I'll track this in a TODO and get around to it _sometime_.